css/css-color/predefined-016.html

```diff
@@ -4,11 +4,11 @@
 <link rel="author" title="Chris Lilley" href="mailto:chris@w3.org">
 <link rel="help" href="https://drafts.csswg.org/css-color-4/#predefined">
 <link rel="match" href="greensquare-090-ref.html">
-<meta name="assert" content="Color function with explicit XYZ value matches sRGB #009900">
+<meta name="assert" content="Color function with explicit xyz-d50 value matches sRGB #009900">
 <style>
     .test { background-color: red; width: 12em; height: 6em; margin-top:0}
     .ref { background-color: #009900; width: 12em; height: 6em; margin-bottom: 0}
-    .test {background-color: color(xyz 0.12266 0.22836 0.03093)}
+    .test {background-color: color(xyz-d50 0.12266 0.22836 0.03093)}
```

Comment:
Should there be a new test testing xyz-d65/xyz then?

Reply:
There are a number of other tests that cover xyz-d65/xyz (see xyz-001.html, xyz-002.html, etc. in the same directory).